qlib/log.py

```diff
@@ -112,6 +108,27 @@ def set_log_with_config(log_config: dict):
     logging_config.dictConfig(log_config)
 
 
+def set_log_basic_config(filename: Optional[Text] = None, format: Optional[Text] = None, level: Optional[int] = None):
```

Comment:
Most of the loggers used in `Qlib` are initialized through the function `get_module_logger`. If you want to do some enhancement about setting the `logger`, you may want to make some modifications upon that function instead of creating a new one. This function seems to be a little bit redundant.

Reply:
The motivation is that `get_module_logger` is called in different separate Python files, and it is hard for users to control the detailed behavior of `get_module_logger` function in each file.
This function provides a global control for logging, a case is that before each experiment runs, we can use this function to save the following logs into a file under the recorder directory (without caring about the kwargs of get_module_logger in each file).